.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           init-tools: 'true'
 
   lint:
-    needs: [ init-hermit ]
+    #needs: [ init-hermit ]
```

Comment:
maybe delete this instead of comment out?

Reply:
Done